### ISSUE TITLE
[BUG]: Weird Red Mesh

### ISSUE THREAD

**Issue by ascariis-sys** [typical-developers/oaklands-feedback#1175]
### Roblox Username

ascariis

### What version did this issue occur in?

1.69.2

### Describe the issue in detail.

A red mesh appears in various parts of the lumite caves . Only visible from one side and collision is disabled

### Provide some images / videos of the issue in action.

https://github.com/user-attachments/assets/6a008378-3596-495d-92da-12286ef7f6f5

![image-254](https://github.com/user-attachments/assets/d0e79b6d-d660-4145-a4e9-9317f07891f8)

### Before Submitting

- [X] I have confirmed that nobody else reported this issue.
- [X] The details of this report are legitimate.
- [X] I understand that I cannot be reimbursed here for in-game content I have lost.

**Comment by PoseidonFAN1** [comment]
Yes, I've noticed this as well. In many parts of the Lumite caves and also some parts of the desert.
